app/src/ui/lib/list/list.tsx

```diff
@@ -729,6 +735,9 @@ export class List extends React.Component<IListProps, IListState> {
     }
     this.scrollToRow = -1
 
+    const scrollTop =
+      this.props.setScrollTop !== undefined ? this.props.setScrollTop : 0
```

Comment:
Ah, that is definitely correct. I have tested now with undefined as the value to confirm the behavior is as expected, and it seems it is fine to set `scrollTop` directly to the props value.

I had originally set it up this way because I was having some inconsistent behavior and was trying to eliminate potential causes. It was not this section causing any issues. I will change this to simply use the original props value.